tuning/sft_trainer.py

```diff
@@ -200,6 +200,8 @@ def train(
         ),
         cache_dir=train_args.cache_dir,
         use_fast=True,
+        legacy=False,
+        padding_side="right",
```

Comment:
Move this into:
```
tokenizer.padding_side = "right"
```
after this statement. But I'm not 100% sure if we want right or left..

Reply:
Yeah I think what I put is not correct, most base models have a tokenizer_config.json file, which indicates the padding side used during training, so I think it should default to that
https://huggingface.co/ibm-granite/granite-8b-code-instruct-128k/blob/main/tokenizer_config.json